scalardb-sql-jdbc-sample/README.md

```diff
@@ -1,156 +1,166 @@
-In this tutorial, you will build the application on Cassandra.
-Although Cassandra does not provide ACID transaction capabilities, Cassandra can support ACID transactions by interfacing through ScalarDB SQL (JDBC).
-Please note that application-specific error handling, authentication processing, and similar functions are not included in the sample application, as the focus is on demonstrating the use of ScalarDB SQL (JDBC).
-For detailed information on exception handling in ScalarDB SQL (JDBC), see [Handle SQLException](https://github.com/scalar-labs/scalardb-sql/blob/main/docs/jdbc-guide.md#handle-sqlexception).
+We recommend using the LTS versions mentioned above, but other non-LTS versions may work.
 
-### Schema
+In addition, other JDKs should work with ScalarDB, but we haven't tested them.
+{% endcapture %}
 
-[The schema](schema.sql) is as follows:
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-```sql
-CREATE NAMESPACE IF NOT EXISTS sample;
+In addition, you need access to the [ScalarDB SQL repository on GitHub](https://github.com/scalar-labs/scalardb-sql) and the [packages in the ScalarDB SQL repository](https://github.com/orgs/scalar-labs/packages?repo_name=scalardb-sql), which are private. The packages and repository are available only those who are using ScalarDB Enterprise. If you need a license for ScalarDB Enterprise, please [contact us](https://scalar-labs.com/contact_us/).
 
-CREATE TABLE IF NOT EXISTS sample.customers (
-  customer_id INT PRIMARY KEY,
-  name TEXT,
-  credit_limit INT,
-  credit_total INT
-);
+After confirming that you have access to the ScalarDB SQL repository and its packages, you will need to set your GitHub username and your personal access token. To specify these properties, you can do one of the following:
 
-CREATE TABLE IF NOT EXISTS sample.orders (
-  customer_id INT,
-  timestamp BIGINT,
-  order_id TEXT,
-  PRIMARY KEY (customer_id, timestamp)
-);
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Gradle_properties_via_command', 'tabset-1')" id="defaultOpen-1">Add to Gradle properties via Terminal</button>
+  <button class="tablinks" onclick="openTab(event, 'Environment_variables', 'tabset-1')">Add as environment variables</button>
+</div>
 
-CREATE INDEX IF NOT EXISTS ON sample.orders (order_id);
+<div id="Gradle_properties_via_command" class="tabcontent" markdown="1">
 
-CREATE TABLE IF NOT EXISTS sample.statements (
-  order_id TEXT,
-  item_id INT,
-  count INT,
-  PRIMARY KEY (order_id, item_id)
-);
+Specify the properties with the `-P` option by running the `./gradlew` command as follows, replacing `<YOUR_GITHUB_USERNAME>` with your GitHub username and `<YOUR_PERSONAL_ACCESS_TOKEN>` with your personal access token:
 
-CREATE TABLE IF NOT EXISTS sample.items (
-  item_id INT PRIMARY KEY,
-  name TEXT,
-  price INT
-);
+```console
+$ ./gradlew run ... -Pgpr.user=<YOUR_GITHUB_USERNAME> -Pgpr.key=<YOUR_PERSONAL_ACCESS_TOKEN>
```

Comment:
Running this command with `...` results in Gradle failing to build. Since users are likely to copy and paste this command, is it OK if we remove `...`?

Reply:
I think `...` means arbitrary parameters. So replacing it with `<PARAMETERS>` or something might be clearer?

Reply:
Ah, that would be helpful to note instead. Thank you! 

Gradle seems to refer to those parameters as [options](https://docs.gradle.org/current/userguide/command_line_interface.html), so I've added that here.